tests/test_fyle/fixtures.py

```
@@ -651,6 +654,168 @@
+            'report_id': 'rpQE6alQtnM1',
+            'report_title': '4 same: multiple line items',
+            'settled_at': None,
+            'source_account_type': 'PERSONAL_CORPORATE_CREDIT_CARD_ACCOUNT',
+            'spent_at': datetime.datetime(2024, 11, 15, 0, 0),
+            'state': 'APPROVED',
+            'sub_category': None,
+            'tax_amount': None,
+            'tax_group_id': None,
+            'vendor': 'Dominos Pizza',
+            'verified_at': None,
+            'bank_transaction_id': 'btxnQe02cfpD9n'
```

Ruff

Dictionary key literal `'bank_transaction_id'` repeated

(F601)